README.md

```diff
@@ -1,9 +1,67 @@
-# Jarivs Data Engineering Training
-1. [Linux cluster monitoring agent (Linux and SQL)](./linux_sql) Completed
-2. [Core Java Apps](./core_java) Completed
-3. [Springboot Trading REST API](./springboot) Completed
-4. [Python Data Analytics](./python_data_analytics) Completed
-5. [Hadoop/Hive](./hadoop) In-progress
-6. [Spark/Scala](./spark) In-progress
-7. [Cloud/DevOps](./cloud_devops) In-progress
+# Habib Mohamed . Jarvis Consulting
 
+I am a University of Windsor graduate in Computer Science. I have worked in various positions as a developer working on front-end, data, and website development. I like to figure out solutions to challenging problems, and to come up with unorthodox means to implement stuff.
+
+## Skills
+
+**Proficient:** Java, Linux/Bash, RDBMS/SQL, Agile/Scrum, Git, HTML/CSS, Docker
+
+**Competent:** Python, C/C++/C#, Spring, JUnit, Flutter, PHP, JavaScript, Maven
```

Comment:
move C/C++/C# to the end 

**Proficient:** Java, Sprintboot, Python, Linux/Bash, RDBMS/SQL, Agile/Scrum, Git, Docker

**Competent:** C/C++/C#, Flutter, PHP, JavaScript, HTML/CSS 

You will add more skills when you complete more projects